test/unit/reductions/exponentiated_gradient/simple_learners.py

```diff
@@ -34,3 +34,19 @@ def fit(self, X, Y, sample_weight):
 
     def predict(self, X):
         return X.dot(self.weights)
+
+
+class MockEstimator:
+    def fit(self, X, y, sample_weight):
+        pass
+
+    def predict(self, X):
+        return np.ones(X.shape[0])
+
+
+class MockSKLearnEstimator(MockEstimator):
```

Comment:
Not sure why you need these, but you probably want to inherit from `BaseEstimator` if this is supposed to be `sklearn` compatible.

Reply:
Yes, thanks for pointing this out. Will just remove this dummy class and use `BaseEstimator` directly!